src/main/kotlin/thatline/localup/chatgpt/service/ChatGptService.kt

```
@@ -41,26 +52,200 @@ class ChatGptService(
+        - 절대로 마크다운(**bold**, ##제목, - 목록 등)을 사용하지 마세요.
+        - 각 섹션마다 빈 줄을 추가하여 단락을 명확히 구분하세요.
+        - 긴 문장은 적절한 위치에서 줄바꿈하여 읽기 쉽게 만드세요.
+        - 번호 목록: 1. 2. 3. 형식 사용
+        - 하위 항목: - 또는 · 기호 사용
+        - 제목이나 강조가 필요한 부분은 "대괄호 [강조내용]" 형식 사용
     """.trimIndent()
+
+    private fun getBusinessInfo(userId: String): FindBusinessDto? {
+        return try {
+            userService.findBusiness(userId)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)